code/__DEFINES/machines.dm

```diff
@@ -43,6 +43,14 @@
 #define SHOCK (1<<3)
 #define SAFE (1<<4)
 
+//defines to be used with the door's open()/close() procs in order to discriminate what type of open is being done. The door will never open if it's been physically disabled (i.e. welded, sealed, etc.).
+/// We should go through the door's normal opening procedure, no overrides.
+#define DOOR_DEFAULT_CHECKS 0
+/// We're not going through the door's normal opening procedure, we're forcing it open. Can still fail if it's emagged or something. Costs power.
+#define DOOR_FORCED_CHECKS 1
+/// We are getting this door open if it has not been physically held shut somehow. Play a special sound to signify this level of opening.
+#define DOOR_BYPASS_CHECKS 2
```

Comment:
Maybe I am being overly picky here because you do get the explanation from the doc comments but `forced` and `bypass` feel like basically synonyms to me, it would make an equal amount of sense if the names were swapped.

Would something like default / forced / pried work? It _is_ kind of hard to think of a word differentiating these two behaviours...

Reply:
i renamed it to be a bit less synonymous